bundles/binding/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/VeluxItemType.java

```diff
@@ -112,7 +106,8 @@
     private boolean itemIsWritable;
     private boolean itemIsExecutable;
 
-    private static Logger logger = LoggerFactory.getLogger(VeluxItemType.class);
+    private final Logger logger = LoggerFactory.getLogger(VeluxItemType.class);
```

Comment:
The `logger_` variable should be removed, and all references to it converted back to references to `logger`.

I didn't notice before that this class contains a `static` method.  If that method needs to remain static, then it's ok to leave the logger as static also.